lib/nokogiri/xml/node.rb

```diff
@@ -192,16 +198,57 @@ def decorate!
 
       # :section: Manipulating Document Structure
 
-      ###
-      # Add +node_or_tags+ as a child of this Node.
-      #
-      # +node_or_tags+ can be a Nokogiri::XML::Node, a ::DocumentFragment, a ::NodeSet, or a String
-      # containing markup.
-      #
-      # Returns the reparented node (if +node_or_tags+ is a Node), or NodeSet (if +node_or_tags+ is
-      # a DocumentFragment, NodeSet, or String).
-      #
-      # Also see related method +<<+.
+      # :call-seq:
+      #   add_child(object) -> node or nodeset
+      #
+      # Appends child nodes to +self+.
+      #
+      # Argument +object+ is one of:
```

Comment:
Ugh, reading the previous documentation, I'm now realizing that object could _also_ be a DocumentFragment